wfe2/wfe.go

```diff
@@ -1540,6 +1555,16 @@ func (wfe *WebFrontEndImpl) Certificate(ctx context.Context, logEvent *web.Reque
 		return
 	}
 
+	if request.Method == "GET" && wfe.clk.Since(cert.Issued) < wfe.staleTimeout {
```

Comment:
This and below seems like they should also be checking `request.Url.Path` for the `getXXX` variants, otherwise GET requests to the original ACME endpoints will also be constrained by this staleness requirement.